docs/additional-configuration.adoc

```diff
@@ -211,7 +212,8 @@ EOF
 kubectl create secret -n "$NAMESPACE" generic git-ssh-key \
   --from-file=dwo_ssh_key="$SSH_KEY" \
   --from-file=dwo_ssh_key.pub="$SSH_PUB_KEY" \
-  --from-file=ssh_config=/tmp/ssh_config
+  --from-file=ssh_config=/tmp/ssh_config \
+  --from-literal=passphrase="$PASSPHRASE"
```

Comment:
I have tried the command without the `PASSPHRASE` env variable and it worked ok, but the `passphrase` key of the secret data has no value: 
```yaml
data:
  dwo_ssh_key: ***
  dwo_ssh_key.pub: ***
  passphrase: ''
  ssh_config: ***
```
Such empty keys will not be mounted to the container file system as files and if the `ssh-askpass.sh` is executed it will throw an error about missing passphrase file.

Reply:
Thanks for checking. Just to confirm: if the user's SSH key does _not_ have a passphrase, will `ssh-askpass.sh` be invoked? I imagine it won't, based on it's documentation:

`If ssh-add **needs** a passphrase, it will read the passphrase from the current terminal if it was run from a terminal...`

The concern is just making sure that users are able to copy & paste the instructions and have them work, even if their SSH key does not have a passphrase (though their secret will have an empty passphrase data field).

Reply:
> if the user's SSH key does not have a passphrase, will `ssh-askpass.sh` be invoked?

I have tried the flow with SSH key with an empty passphrase, and neither the default askpass scriptn nore the provided `ssh-askpass.sh` script was not invoked.

Reply:
Perfect!